### PR TITLE
Add MiKTeX

### PR DESCRIPTION
**Pull request by nicklauslittle-gov**

### CODE REVIEW

**Comment by nicklauslittle-gov** [comment]
There may be an issue with this. Will update when this is good to go.

File: src/pkgs/miktex.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+$global:PwrPackageConfig = @{
+	Name = 'python'
```
Comment (troyaf) [inline src/pkgs/miktex.ps1:2]: Update name

File: src/pkgs/miktex.ps1 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+function global:Install-PwrPackage {
+	$Params = @{
+		Owner = 'MiKTeX'
+		Repo = 'miktex'
+		TagPattern = '^([0-9]+)(\.([0-9]+)(\.([0-9]+)(\.([0-9]+))?)?)?$'
```
Comment (troyaf) [inline src/pkgs/miktex.ps1:9]: Make this 3 capturing groups for `major.minor.patch`

File: src/pkgs/miktex.ps1 [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	Start-Process $Asset -ArgumentList "--auto-install=yes --unattended --private --user-install=$ToolDir --no-registry" -Wait
+	Write-PackageVars @{
+		env = @{
+			path = (Get-ChildItem -Path '\pkg' -Recurse -Include 'latex.exe' | Select-Object -First 1).DirectoryName
+		}
+	}
+}
+
+function global:Test-PwrPackageInstall {
+	Get-Content '\pkg\.pwr'
```
Comment (troyaf) [inline src/pkgs/miktex.ps1:31]: Mimic some of the other packages with some usage of the `exe`, like `-version` or something

**Comment by troyaf** [comment]
BTW If you make the branch name the same as the `.ps1`, then the workflow will only run for this tool.

**Comment by nicklauslittle-gov** [comment]
I'm closing this so the branch will match the added package.

**Pull request closed without merging**